forest/cli/src/cli/info_cmd.rs

```diff
@@ -0,0 +1,223 @@
+// Copyright 2019-2023 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use std::time::Instant;
+
+use clap::Subcommand;
+use colored::*;
+use forest_blocks::tipset_keys_json::TipsetKeysJson;
+use forest_cli_shared::{cli::CliOpts, logger::LoggingColor};
+use forest_rpc_client::{
+    chain_get_name, chain_get_tipset, chain_head, start_time, wallet_default_address,
+};
+use forest_shim::econ::TokenAmount;
+use forest_utils::io::parser::{format_balance_string, FormattingMode};
+use fvm_shared::{clock::EPOCH_DURATION_SECONDS, BLOCKS_PER_EPOCH};
+use time::OffsetDateTime;
+
+use super::Config;
+use crate::cli::handle_rpc_err;
+
+#[derive(Debug, Subcommand)]
+pub enum InfoCommand {
+    Show,
+}
+
+#[derive(Debug, strum_macros::Display)]
+enum SyncStatus {
+    Ok,
```

Comment:
Do we have any user-facing documentation that explains `Ok` is within 1.5 epochs, `Slow` is within 5 epochs and `Behind` is >5 epochs after `HEAD`?

Reply:
Nope, lotus impl doesn't have any docs for that: https://github.com/filecoin-project/lotus/blob/76776e5cc1ce288e7fa6c60980949ed74633f24f/cli/info.go#L215 Neither i could find anything in filecoin spec.